src/pages/settings/Security/AddDelegate/DelegateMagicCodeModal.tsx

```diff
@@ -14,13 +14,14 @@ import ROUTES from '@src/ROUTES';
 type DelegateMagicCodeModalProps = {
     login: string;
     role: ValueOf<typeof CONST.DELEGATE_ROLE>;
+    isValidateCodeActionModalVisible: boolean;
+    setIsValidateCodeActionModalVisible: (isValidateCodeActionModalVisible: boolean) => void;
     onClose?: () => void;
 };
 
-function DelegateMagicCodeModal({login, role, onClose}: DelegateMagicCodeModalProps) {
+function DelegateMagicCodeModal({login, role, onClose, isValidateCodeActionModalVisible, setIsValidateCodeActionModalVisible}: DelegateMagicCodeModalProps) {
```

Comment:
Can we use a single `onClose` prop here, and not pass the `setIsValidateCodeActionModalVisible` ? By pressing onClose, we can handle the `setIsValidateCodeActionModalVisible` on the `ConfirmDelegatePage`

Reply:
@cretadn22 ⬆️